Makefile

```diff
@@ -11,9 +11,10 @@ aws-ip-ranges:
 	curl -o aws-ip-ranges.json https://ip-ranges.amazonaws.com/ip-ranges.json
 
 npm-setup:  # runs all front-end setup
-	npm install
+	npm ci
 	npm run build | grep -v "node_modules\|\[built\]"
 	npm run build-scss
+	git status
```

Comment:
`git status` might show changes we don't care about, though maybe we could arrange to checkpoint its output before and compare it after to see if something happened anew. That's likely to be a pain to do, so I'd just leave it out for now.